contributing/process-outline.md

```diff
@@ -18,6 +18,7 @@ interested in working on.
 ## 2. Describe the Acceptance Criteria
 
 - Before starting to work on the issue, describe the acceptance criteria (AC). 
+- Write the AC in the issue when you create one.
```

Comment:
let's put it in the same bullet point (essentially merge this new one with the one above)

Reply:
Done